src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.PivotDaoImpl;
+import core.basesyntax.dao.TransactionDaoImpl;
+import core.basesyntax.service.BalanceService;
+import core.basesyntax.service.BalanceServiceImpl;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        BalanceService balanceService = new BalanceServiceImpl(new TransactionDaoImpl(),
+                                            new PivotDaoImpl(), new OperationStrategyImpl());
+
```

Comment:
you don't need several daos in this task, only one that will be updating your fruits storage but you don't have one, so you don't need this layer